text/0000-light-dom.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Light DOM Component
+status: DRAFTED
+created_at: 2020-09-10
+updated_at: 2020-09-10
+champion: Philippe Riand (priand), Ted Conn (tconn)
+pr: https://github.com/salesforce/lwc-rfcs/pull/44
+---
+
+# RFC: Light DOM Component
+
+## Summary
+
+As of today, all the LWC components inheriting from `LightningElement` render their content to the shadow DOM. This proposal introduces a new kind of component which renders its content as children in the Light DOM.
+
+## Basic example
+
+When the Shadow DOM is turned off for a component, its content is not attached to its shadow-root, but to the element itself. Here is an example, showing whenever Shadow DOM is on or off:
+
+_Shadow DOM_
+
+```html
+<app-container-blue-shadow>
+  #shadow-root (open)
+  | <div>
+  |   <b>Blue Shadow:</b>
+  |   <span class="counter">...</span>
+  |   <button type="button">Add one</button>
+  | </div>
+</app-counter-blue-shadow>
+```
+
+_Light DOM_
+
+```html
+<app-container-blue-light>
+    <div>
+      <b>Blue Light:</b>
+      <span class="counter">...</span>
+      <button type="button">Add one</button>
+    </div>
+</app-counter-blue-light>
+```
+
+As a result, when the content of a component resides in the Light DOM, it can be accessed like any other content in the Document host, and thus behave like any other content (styling, APIs, accessibility, third party tooling...).
+
+## Motivation
+
+Consumer applications require DOM traversal and observability of an application’s anatomy from the document root. Without this, theming becomes hard and 3rd party applications do not run properly:
+
+- **Theming and branding**
```

Comment:
As Dean said in DRB today, UIP has the same (near-term future) requirements around theming and branding for LEX and LWR apps as Communities.

Reply:
I don't believe the LEX requirements are the same as what Communities is requesting because the number of actors in the system is very different. Let's discuss offline.

Reply:
@hsterlingsfdc Could you make sure to bring the LWC team into the loop when this will be discussed/designed? I would like to better understand the LEX and LWR requirements in this regard.